code/game/mecha/mecha_construction_paths.dm

```diff
@@ -863,6 +863,76 @@
 	feedback_inc("mecha_honker_created",1)
 	return
 
+/datum/construction/mecha/reticence_chassis
+	steps = list(list("key"=/obj/item/mecha_parts/part/reticence_torso),//1
+					 list("key"=/obj/item/mecha_parts/part/reticence_left_arm),//2
+					 list("key"=/obj/item/mecha_parts/part/reticence_right_arm),//3
+					 list("key"=/obj/item/mecha_parts/part/reticence_left_leg),//4
+					 list("key"=/obj/item/mecha_parts/part/reticence_right_leg),//5
+					 list("key"=/obj/item/mecha_parts/part/reticence_head)
+					)
+
+/datum/construction/mecha/reticence_chassis/action(atom/used_atom,mob/user)
+	return check_all_steps(used_atom,user)
+
+/datum/construction/mecha/reticence_chassis/custom_action(step, atom/used_atom, mob/user)
+	user.visible_message("[user] has connected [used_atom] to the [holder].", "<span class='notice'>You connect [used_atom] to the [holder].</span>")
+	holder.overlays += used_atom.icon_state+"+o"

```

Comment:
tbh I'd file this under  "the code is fucked up but that's beyond the scope of this PR, which isn't making things worse."